string id
bool is_time
---
CustomSignal signal